Assets/MixedRealityToolkit.Tests/PlayModeTests/FocusProviderTests.cs

```diff
@@ -384,6 +385,36 @@ public IEnumerator TestPrioritizedLayerMask()
             Assert.IsNull(shellHandRayPointer.Result?.CurrentPointerTarget, $"{noPriorityCube.name} should NOT be raycast target by shell hand ray pointer");
         }
 
+        /// <summary>
+        /// </summary>
```

Comment:
please describe what you're testing in the summary

Reply:
Oh weird, i could have sworn I filled this out! I'll update